src/formatter/get_color_fns.ts

```diff
@@ -48,3 +57,15 @@ export default function getColorFns(enabled: boolean): IColorFns {
     }
   }
 }
+
+function detectSupport(
+  stream: Writable,
+  env: NodeJS.ProcessEnv,
+  enabled?: boolean
+): ColorInfo {
+  const support: ColorInfo = supportsColor(stream)
+  if ('FORCE_COLOR' in env || doesNotHaveValue(enabled)) {
```

Comment:
If `FORCE_COLOR` is present, we'll just let the library handle that and not bother with our own option.

Reply:
Maybe that comment would be useful in the code itself?